helm/ngrok-operator/values.yaml

```diff
@@ -77,7 +77,7 @@ serverAddr: ""
 clusterDomain: svc.cluster.local
 
 ## @param apiURL  This is the URL of the ngrok API. You should set this if you are using a custom API URL.
-apiURL: ""
+apiURL: "https://api.ngrok.com/"
```

Comment:
**Non-blocking**: I lean towards just letting this take the default in ngrok-api-go and not explicitly setting it.